src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -344,9 +345,14 @@ function WorkspaceMembersPage(props) {
 
         return result;
     };
-
     const data = getMemberOptions();
-    const headerMessage = searchValue.trim() && !data.length ? props.translate('workspace.common.memberNotFound') : '';
+
+    const getHeaderMessage = () => {
+        if (isOfflineAndNeedGetData) {
```

Comment:
Maybe this variable can be named a bit clearer. How about:

```suggestion
        if (isOfflineAndNoMemberDataAvailable) {
```

Reply:
@MariaHCD Thanks for your comment, i just updated code. Please check again